rocAL/source/parameters/parameter_factory.cpp

```diff
@@ -104,33 +104,46 @@ void ParameterFactory::generate_seed() {
     _seed = rd();
 }
 
+int64_t
```

Comment:
I can see the seed changes here as well. Please remove these if it was unintentional

Reply:
Added them as the seed changes are also related to the parameterVX changes

Reply:
Even though files overlap, only retain the changes which pertains to the parameter vx

If not add all seed related changes in this PR and also mention the same in the PR description

Reply:
Address this

Reply:
Updated the PR name and description for including both paramVX and seedsequence changes.